jwst/datamodels/ifuimage.py

```diff
@@ -35,8 +32,14 @@ class IFUImageModel(DataModel):
     wavelength : numpy float32 array
          wavelength
 
-    pathloss : numpy float32 array
-         pathloss correction
+    pathloss_ps : numpy float32 array
+         pathloss correction for point source
+
+    pathloss_un : numpy float32 array
+         pathloss correction for uniform source
+
+    area : numpy float32 array
```

Comment:
Note to reviewers: the `area` object just moved in the order, so that it shows up after all the other extensions in the actual FITS files (that's just my OCD kicking in).

Reply:
I think names like `pathloss_point` and `pathloss_uniform` would be a bit more descriptive.

Reply:
Agreed. I wanted to keep the EXTNAME values short, but the data array names could be spelled out more fully.

Reply:
Fixed in latest commit.